Cargo.toml

```diff
@@ -33,6 +33,7 @@ criterion = "0.3.1"
 rand = "0.7"
 async-std = { version = "1.5", features = ["attributes", "unstable"] }
 futures = { version = "^0.3", features = ["std"] }
+waker-fn = "1.1.0"
```

Comment:
optional = true?

Reply:
Nvm, I see it's in dev-dependencies

Reply:
It's a dev dependency so you can't do that